app/auth/views/api_to_cookie.py

```diff
@@ -0,0 +1,33 @@
+import arrow
+from flask import redirect, url_for, request, flash
+from flask_login import current_user, login_user
+
+from app.auth.base import auth_bp
+from app.log import LOG
+from app.models import ApiToCookieToken
+from app.utils import sanitize_next_url
+
+
+@auth_bp.route("/api_to_cookie", methods=["POST"])
+def api_to_cookie():
+    if current_user.is_authenticated:
+        LOG.d("user is already authenticated, redirect to dashboard")
+        return redirect(url_for("dashboard.index"))
+    body = request.json
```

Comment:
This is the url that app should open in the webview right? Do you know if the app can open a webview with a POST request? This URL is also used by the web extension and opening an URL with a POST request requires workaround like https://stackoverflow.com/a/17089124

Reply:
Ah. Good catch. I've changed it to a `GET` request :+1: